### PR TITLE
Update production.md

### PR DESCRIPTION
**Pull request by zomars**
While following this guide I'm receiving warnings that the production mode is not set. I found it odd that the mode was removed in sections following the Setup section. Is this intended?

### CODE REVIEW

File: src/content/guides/production.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ __webpack.prod.js__
   const common = require('./webpack.common.js');
 
   module.exports = merge(common, {
+    mode: 'production',
 +   devtool: 'source-map',
     plugins: [
 -     new UglifyJSPlugin()
```
Comment (Legends) [inline src/content/guides/production.md]: You have to; `-require(UglifyJSPlugin)` too.

Comment (Legends) [reply src/content/guides/production.md]: It's not related to your PR, but lines 204-205 are strange...

Comment (zomars) [reply src/content/guides/production.md]: That's how the docs are presenting code changes from the previous step, it just looks weird on GitHub. 

captura de pantalla 2018-06-20 a las 15 05 59

Comment (zomars) [reply src/content/guides/production.md]: Also `UglifyJSPlugin` is already required on lines 168 and 195.

Comment (zomars) [reply src/content/guides/production.md]: Technically it's not removing brackets, it's just adding a comma. That's how git sees it. If you add a comma git takes the whole line as modified.

Comment (Legends) [reply src/content/guides/production.md]: Ah ok, it means, replace this with that.  You are right. I think it's enough for today ;-)

**Comment by montogeek** [comment]
Thanks!

**Comment by zomars** [comment]
Thank you @montogeek ! Should I add myself as a contributor?

**Comment by montogeek** [comment]
@zomars yes

**Comment by zomars** [comment]
Should I make a new PR for that? Excuse my ignorance

**Comment by montogeek** [comment]
@zomars Yes please

**Pull request merged**